### PR TITLE
Update: Baidunetdisk 7.6.0

### PR DESCRIPTION
**Pull request by Voya9e**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/24197)

### CODE REVIEW

**Review by ImJoakim — COMMENTED** [review]
You can only submit 1 manifest per PR. You are currently trying to add version `7.5.1` and version `7.6.0`. Based on your PR title, you need to remove the files for version `7.5.1`.

File: manifests/b/baidu/baidunetdisk/7.6.0/baidu.baidunetdisk.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+Installers:
+- InstallerLocale: zh-CN
+  Architecture: x86
+  InstallerType: nullsoft
+  Scope: user
+  InstallerUrl: https://issuecdn.baidupcs.com/issue/netdisk/yunguanjia/BaiduNetdisk_7.6.0.13.exe
+  InstallerSha256: d7f565b75e921380dfb66087354baf9aa920a9b1398f0af8865b7388227076d0
+  InstallerSwitches:
+    Silent: /S
+    SilentWithProgress: /S
```
Comment (OfficialEsco) [inline manifests/b/baidu/baidunetdisk/7.6.0/baidu.baidunetdisk.installer.yaml:21]: This is not needed
```suggestion
```

File: manifests/b/baidu/baidunetdisk/7.6.0/baidu.baidunetdisk.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+PackageIdentifier: baidu.baidunetdisk
+PackageVersion: 7.6.0
+PackageLocale: en-US
+Publisher: 北京度友科技有限公司
+PublisherUrl: https://www.baidu.com/
+PublisherSupportUrl: http://yun.baidu.com/disk/help
+PrivacyUrl: http://yun.baidu.com/disk/privacy
+Author: Baidu
+PackageName: baidunetdisk
```
Comment (OfficialEsco) [inline manifests/b/baidu/baidunetdisk/7.6.0/baidu.baidunetdisk.locale.en-US.yaml:12]:
```suggestion
PackageName: 百度网盘
```

File: manifests/b/baidu/baidunetdisk/7.6.0/baidu.baidunetdisk.locale.zh-CN.yaml [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+PackageIdentifier: baidu.baidunetdisk
+PackageVersion: 7.6.0
+PackageLocale: zh-CN
+Publisher: 北京度友科技有限公司
+PublisherUrl: https://www.baidu.com/
+PublisherSupportUrl: http://yun.baidu.com/disk/help
+PrivacyUrl: http://yun.baidu.com/disk/privacy
+Author: 北京度友科技有限公司
+PackageName: BaiduNetdisk
```
Comment (OfficialEsco) [inline manifests/b/baidu/baidunetdisk/7.6.0/baidu.baidunetdisk.locale.zh-CN.yaml:12]:
```suggestion
PackageName: 百度网盘
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]

**Review by OfficialEsco — DISMISSED** [review]
Also accepting this one since the zh-CN locale got the correct PackageName

**Review by Voya9e — COMMENTED** [review]
> Also accepting this one since the zh-CN locale got the correct PackageName

But I have changed the PackageName in the zh-CN locale.
I can't understand.

**Comment by OfficialEsco** [comment]
> But I have changed the PackageName in the zh-CN locale.
> I can't understand.

Yes you did in the zh-CN locale, but not the en-US locale.
The Software installs as `百度网盘` on a en-US computer, which is why i suggested this change https://github.com/microsoft/winget-pkgs/pull/24197#discussion_r684798516
Atm the en-US locale PackageName is more of a English moniker for the software, but since its correct in the zh-CN locale this is "ok", however someone will probably update it to `百度网盘` in the next version.

**Pull request merged**